python/cucim/setup.py

```diff
@@ -31,13 +28,11 @@ def read(*names, **kwargs):
     ) as fh:
         return fh.read()
 
-# lib_paths = list(map(lambda x: basename(x), glob("src/cucim/clara/*.so*")))
-
 
 opts = dict(
     name='cucim',
     version=read('VERSION').strip(),  # versioneer.get_version(),
-    # cmdclass=versioneer.get_cmdclass()
+    cmdclass=versioneer.get_cmdclass(),
```

Comment:
If we are using versioneer for `cmdclass`, should we use for `version` as well?

Reply:
In case of versioneer, version comes from git tags, but C++ code's build system currently requires version as `VERSION` file.
I can try to refactor C++ code builder to get the version from git tags later (https://ipenguin.ws/2012/11/cmake-automatically-use-git-tags-as.html) but the current VERSION file seems okay to me.

Currently, version number from versioneer is used only when CuPy with scikit-image is available (https://github.com/rapidsai/cucim/issues/44, https://github.com/rapidsai/cucim/pull/48/commits/818dc2f00416aa7ec07aabc27c460948661ce510#diff-80a85bd6c52ce89c0dc59a606ca1205a2673c20ff860c65a3964db51ffa64ec8R47)